docs/source/api/power-board.rst

```diff
@@ -0,0 +1,109 @@
+   pressed.
+
+.. code:: python
+
+   r.power_board.outputs.power_off()
+   r.power_board.outputs.power_on()
+
+
+You can also get information about and control each output in the group.
+An output is indexed using the appropriate ``PowerOutputPosition``.
+
+.. code:: python
+
+   from sbot import PowerOutputPosition
+
+   r.power_board.outputs[PowerOutputPosition.H0].is_enabled = True
+   r.power_board.outputs[PowerOutputPosition.L3].is_enabled = False
+   
+   boolean_value = r.power_board.outputs[PowerOutputPosition.L2].is_enabled
+
+   current_amps = r.power_board.outputs[PowerOutputPosition.H1].current
+
+.. Warning:: The motor and servo boards are powered through these
+   power outputs, whilst the power is off, you won’t be able to control
+   your motors or servos. They will register as a missing board and your code will
+   break if you try and control them.
+
+Battery Sensor
+--------------
+
+The power board has some sensors that can monitor the status of your battery.
+This can be useful for checking the charge status of your battery.
+
+.. code:: python
+   
+   battery_voltage = r.power_board.battery_sensor.voltage
+   battery_current_amps = r.power_board.battery_sensor.current
+
+Buzzing 🐝
+--------
+
+The power board has a piezo buzzer which can buzz.
+
+The ``buzz`` function accepts multiple parameters, depending on what you
+want to play. The first argument is the duration of the beep, in
+seconds. The later arguments are either the note you want to play, or
+the frequency of the buzzer (in Hz). You have to specify which of note
+or frequency you’re passing using a keyword argument, your code will
+fail otherwise.
+
+Theoretically, the piezo buzzer will buzz at any provided frequency,
+however humans can only hear between `20Hz and
+20000Hz <https://en.wikipedia.org/wiki/Hearing_range#Humans>`__.
+
+The ``Note`` Enum provides notes in `Scientific Pitch Notation
+<https://en.wikipedia.org/wiki/Scientific_pitch_notation>`__ between
+``C6`` and ``C8``. You can play other tones by providing a frequency.
+
+.. Hint:: Calling ``buzz`` is non-blocking. Meaning it doesn’t
+   actually wait for the piezo to stop buzzing before continuing with your
+   code. If you want duration to wait for the buzzing to stop, add a
+   ``sleep`` afterwards!
+
+.. code:: python
+
+   from sbot import Note
+
+   # Buzz for half a second in D6.
+   r.power_board.piezo.buzz(0.5, Note.D6)
+
+   # Buzz for 2 seconds at 400Hz
+   r.power_board.piezo.buzz(2, 400)
+
+.. Warning:: Trying to play a note that isn't in the range ``C6`` - ``C8``
+   will range an ``AttributeError``. Try using frequencies instead!
+
+Start Button
+------------
+
+You can manually wait for the start button to be pressed, not only at
+the start.
+
+.. code:: python
+
+   r.power_board.wait_start()
+
+This may be useful for debugging, but be sure to remove it in the
+competition, as your robot can’t be interacted with!
```

Comment:
I don't understand what this paragraph is trying to say.

Reply:
You can't press the start button in the middle of the arena!

Reply:
```suggestion
competition, as you won't be allowed to touch the start button after a match has begun!
```